src/components/Glp/GlpSwap.js

```diff
@@ -973,10 +987,43 @@ export default function GlpSwap(props) {
                           );
                         }
                         return (
-                          <div className="text-white">
-                            {isFeesHigh && <Trans>To reduce fees, select a different asset to receive.</Trans>}
-                            <Trans>Check the "Save on Fees" section below to get the lowest fee percentages.</Trans>
-                          </div>
+                          <>
+                            <StatsTooltipRow
+                              label="Base Fee"
+                              value={formatDeltaUsd(feeItem.deltaUsd, feeItem.bps)}
+                              showDollar={false}
+                              className="text-red"
+                            />
+                            <StatsTooltipRow
+                              label="Max Bonus Rebate"
+                              value={`+${formatUsd(maxRebateUsd.abs())}`}
```

Comment:
using `abs` looks a bit inconsistent

here https://github.com/gmx-io/gmx-interface/pull/674/files#diff-cd0741d6fc8d0943f840d44461016a3ff764a177a6d6da1ab44f38859d4d65d3R976 `swapUsdMin.mul(-1)` is used

and here https://github.com/gmx-io/gmx-interface/pull/674/files#diff-cd0741d6fc8d0943f840d44461016a3ff764a177a6d6da1ab44f38859d4d65d3R979 just `swapUsdMin`. i guess it's better to use `swapUsdMin.mul(-1)` in both cases for simplicity?

Reply:
`formatUsd(maxRebateUsd.abs())` is used to display the rebate reward amount. So I used `abs` to make sure it will always be positive.

I have updated the use of `swapUsdMin` to be consistent now.